mitiq/pec/representations/biased_noise.py

```diff
@@ -81,10 +84,10 @@ def represent_operation_with_local_biased_noise(
         QPROGRAM, followed by a single final biased noise channel, is
         physically implementable.
     """
-    circ, in_type = convert_to_mitiq(ideal_operation)
-
+    circuit_copy = copy.deepcopy(ideal_operation)
+    converted_circ, _ = convert_to_mitiq(circuit_copy)
```

Comment:
Yes, we need them because we have to keep the original circuit intact to pass it through the noise scaling converter. We also need a Mitiq circuit to access the qubits for constructing the representations.

Reply:
I see, thanks!